ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/CommandTopic.java

```diff
@@ -86,7 +89,9 @@ public void start() {
       for (ConsumerRecord<byte[], byte[]> record : iterable) {
         try {
           backupRecord(record);
-        } catch (final Exception e) {
+        } catch (final KsqlException e) {
```

Comment:
again - this should be a very specific exception indicating a deserialization error. Also, why are we even trying to deserialize the command from the backup logic (cc @spena)? It doesn't look like we're using any of the fields.

Reply:
Ok after talking it over with @spena , we don't need this deserialization logic in the backup impl, it should just backup whatever is found in the command topic, deserialization should be handled after it's backed up. So I'll get rid of this catch block and we'll only have the KsqlServerException one.